examples/edit.cpp

```diff
@@ -0,0 +1,22 @@
+#include <iostream>
+#include <cstdlib>
+#include <string>
+
+#include "vega/dictionary/dictionary.h"
+#include "vega/dicom/file.h"
+
+int main(int argc, char *argv[]) {
+  if (argc < 3){
+    std::cout<<"Usage: edit input_file output_file"<<std::endl;
```

Comment:
Could you add spaces around the `<<` operators?
(no specific rule about this in the style guide, but all code examples have spaces: https://google.github.io/styleguide/cppguide.html)

Reply:
One other note, could you add a comment about how to build this binary? This usage message assumes that the output binary will be named `edit` for instance.

Reply:
Yes, you are right.